flink-core/src/test/java/org/apache/flink/core/fs/FileSystemBehaviorTestSuite.java

```diff
@@ -263,6 +266,21 @@ public void testMkdirsFailsWithExistingParentFile() throws Exception {
         }
     }
 
+    /**
+     * This test is added to make sure that each FileSystem's OutputStream implementation is
+     * repeatedly closable. This was necessary due to the fact that {@link
+     * ObjectMapper#writeValue(java.io.OutputStream, Object)} closes the {@link OutputStream}
+     * internally (see FLINK-26957 for more context).
+     */
+    @Test
+    public void testClosingOutputStreamTwice() throws Exception {
```

Comment:
do we still need this test, if we're not double-closing the resource anymore?

Reply:
fair enough, I thought it would be still valuable. But I removed the commit...